CONTRIBUTING.md

```diff
@@ -30,8 +30,10 @@ To contribute to this repo, you'll use the Fork and Pull model common in many op
 Guide](https://github.com/kubernetes/community/blob/master/contributors/guide/github-workflow.md)
 from Kubernetes.
 
-When your contribution is ready, you can create a pull request. Pull requests are often referred to as "PR". In general, we follow the standard [GitHub pull request](https://help.github.com/en/articles/about-pull-requests) process. Follow the template to provide details about your pull request to the maintainers. It's best to break your contribution into smaller PRs with incremental changes, and include a good description of the changes. 
-We require new unit tests to be contributed with any new functionality added. 
+When your contribution is ready, you can create a pull request. Pull requests are often referred to as "PR". In general, we follow the standard [GitHub pull request](https://help.github.com/en/articles/about-pull-requests) process. Follow the template to provide details about your pull request to the maintainers. 
+1. It's best to break your contribution into smaller PRs with incremental changes, and include a good description of the changes. 
+2. We require new unit tests to be contributed with any new functionality added.
+3. For any new feature or capability, documentation is a MUST. Features will not be announced in releases without documentation on how to use them. This is common practice in any open-sourced repository.
```

Comment:
```suggestion
3. We require each feature to be documented as part of the PR. If certain feature is experimental and not documented it will be announced as a dev preview.
```

@Ssukriti droppint the last line to make this more clear and added that non documented features are dev previews not official releases let me know what you think.